hassio/src/addon-view/config/hassio-addon-config.ts

```diff
@@ -111,7 +83,10 @@ class HassioAddonConfig extends LitElement {
     this.requestUpdate();
   }
 
-  private async _resetTapped(): Promise<void> {
+  private async _resetTapped(ev: CustomEvent): Promise<void> {
+    const button = ev.target as any;
```

Comment:
You might want to change these to `currentTarget`, as target could be one the children of `ha-progress-button`
```suggestion
    const button = ev.currentTarget as any;
```

Reply:
True 👍
I'll do another PR to change all of them.